project/Dependencies.scala

```diff
@@ -341,13 +343,15 @@ object Dependencies {
 
   val Sns = Seq(
     libraryDependencies ++= Seq(
-        "com.github.matsluni" %% "aws-spi-akka-http" % AwsSpiAkkaHttpVersion excludeAll ExclusionRule(
-          organization = "com.typesafe.akka"
-        ), // ApacheV2
-        "software.amazon.awssdk" % "sns" % AwsSdk2Version excludeAll (ExclusionRule(
-          organization = "software.amazon.awssdk",
-          name = "netty-nio-client"
-        ), ExclusionRule(organization = "io.netty")), // ApacheV2
+        "com.github.matsluni" %% "aws-spi-akka-http" % AwsSpiAkkaHttpVersion excludeAll // ApacheV2
+        ExclusionRule(organization = "com.typesafe.akka"),
```

Comment:
A bit weird formatting here. Maybe use `()` for a single ExclusionRule here as well as it is doe below?